apps/aecontract/test/aect_test_utils.erl

```diff
@@ -404,7 +415,13 @@ compiler_cmd(Vsn) ->
         ?SOPHIA_LIMA_FATE -> filename:join([BaseDir, "v4.3.1", "aesophia_cli"])
     end.
 
-aci_json_enabled(Vsn) -> Vsn > ?SOPHIA_ROMA andalso not aci_disabled().
+aci_json_enabled(Vsn) ->
+    case aci_disabled() of
+        true -> no;
+        _  when Vsn < ?SOPHIA_LIMA_FATE -> no;
```

Comment:
Only disable it for SOPHIA_ROMA - the aci lirary handles old compiles well.

Reply:
Should be fixed now